data.json

```
@@ -521,5 +588,23 @@
       "quantile",
       "statistics"
     ]
+  },
+  {
+    "command": "",
+    "description": "Automatic Differentiation in Julia",
+    "kind": "header",
+    "package": "Zygote",
+    "tags": [
+      "gradients"
+    ]
+  },
+  {
+    "command": "df(x) = gradient(x -> 3x^2+ 2x + 1, x)[1]",
+    "description": "The gradient function takes another Julia function f and a set of arguments, and returns the gradient with respect to each argument.",
```

```suggestion
    "command": "gradient(f, args...)",
    "description": "Computes the gradient of f at each argument, returning a tuple.",
```
I think this is simpler and more informative of the return type.

---

This seem like a better description!